dns/nixos.org.js

```diff
@@ -171,7 +172,11 @@ D("nixos.org",
 	TXT("_github-pages-challenge-nixos", "f3a423ba6916e972cfb1e74f82f601"),
 
 	// netlify pages
+	ALIAS("@", "nixos-homepage.netlify.app."),
+	CNAME("common-styles", "nixos-common-styles.netlify.app."),
```

Comment:
Are these CNAMEs (and the ALAIAS record above) going to apply cleanly to netlify (which, IIUC, we're still deploying to with the changes in this PR)?

Reply:
Sorry, there is no plan to apply this to Netlify at all. Will update the PR shortly.